### ISSUE TITLE
Error message for failed sidechain deposits

### ISSUE THREAD

**Issue by CryptAxe** [drivechain-project/bitcoin#23]
If a sidechain deposit fails we need to alert the user and explain why it failed and what they need to do.

**Comment by psztorc** [comment]
Let me add that (unless I've misunderstood) this is specific to this issue here: https://github.com/drivechain-project/docs/blob/master/bip1-hashrate-escrow.md#inconvenient-race-condition

**Comment by CryptAxe** [comment]
Yes that is correct ^

**Comment by ghabs** [comment]
This looks like this has been fixed in sidechaindepositdialog.cpp L87
https://github.com/drivechain-project/bitcoin/blob/mainchainBMM/src/qt/sidechaindepositdialog.cpp#L87

**Comment by CryptAxe** [comment]
@ghabs That's true there are actually error messages now. They are very programmer friendly and not so user friendly though. I'll update the name of this issue, as the error messages just need to be made more clear now.
